molecule/default/playbook.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Converge
+  hosts: all
+  roles:
+    - role: ansible-role-beegfs
+      beegfs_enable:
+        mgmt: "{{ inventory_hostname in groups['cluster_beegfs_mgmt'] }}"
+        admon: false
```

Comment:
probably also want to add `rdma: false`  here and the README.md for completeness

Reply:
I see that in some places you use `beegfs_rdma` and in others `rdma` . It's not clear for me which one to use. would it make sense to make it homogeneous to always use the same and remove the other?

Reply:
Ah, beegfs_rdma is the default value, beegfs_enable.rdma is a user configurable value. Both are useful.

Reply:
ok thanks for the clarification. I have added this commit as you requested
https://github.com/stackhpc/ansible-role-beegfs/pull/26/commits/e436bfdd5b6a6fd00d647be84f9bbe9409787dd8

I haven't modified the main README because I see that `defaults/main.yml` enable rdma by default. I only disable it for the molecule tests as the VMs have no infiniband. Is ok like that? let me know if you want any change